<launch>
  <node pkg="jsk_pcl_ros" type="mask_image_filter" name="mask_image_filter">
    <remap from="~input" to="/tablet/head_rgbd_sensor/depth_registered/points" />
    <remap from="~input/mask" to="rect_to_mask_image/output"/>
    <remap from="~input/camera_info" to="/tablet/head_rgbd_sensor/depth_registered/camera_info" />
  </node>
</launch>
